src/HealthChecks/HealthChecks/src/DefaultHealthCheckService.cs

```diff
@@ -170,13 +170,13 @@ private static void ValidateRegistrations(IEnumerable<HealthCheckRegistration> r
 
         internal static class EventIds
         {
-            public static readonly EventId HealthCheckProcessingBegin = new EventId(100, "HealthCheckProcessingBegin");
-            public static readonly EventId HealthCheckProcessingEnd = new EventId(101, "HealthCheckProcessingEnd");
+            public static readonly EventId HealthCheckProcessingBegin = (100, nameof(HealthCheckProcessingBegin));
```

Comment:
https://github.com/aspnet/AspNetCore/pull/11379#discussion_r295441810

Reply:
Oh, I see what you mean. Although if the shape of logging is considered a contract that shouldn't change - there's probably several details that might be changed accidentally as easily. Using string instead of nameof doesn't seem like a strict enforcement... 

Just thinking out loud, I wonder if there would be a convenient way to validate that logging contract hasn't changed in the unit tests?